_includes/head.html

```diff
@@ -10,4 +10,6 @@
   <link rel="preconnect" href="https://fonts.googleapis.com">
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700;800;900&display=swap" rel="stylesheet">
+<script src="{{ site.baseurl }}/assets/javascript/tracking.js"></script>
+<script src="https://analytics.ossupstream.org/matomo.js" async defer></script>
```

Comment:
We have an `_includes/rh-analytics-head.html` where the existing analytics scripts get bundled into the build. Maybe these new `<script>`s can go in there as well?

Is there a dev/staging env version for this setup, like our old setup has? If not then perhaps it should just be left out in dev builds.

Reply:
Sorry for not noticing this prior. I've moved the <script> from the head file to the rhanalytics-head file and only added it to production.

Reply:
No worries, thanks for the help with this.